Formula/ragel.rb

```diff
@@ -1,21 +1,20 @@
 class Ragel < Formula
   desc "State machine compiler"
   homepage "https://www.colm.net/open-source/ragel/"
-  url "https://www.colm.net/files/ragel/ragel-6.9.tar.gz"
-  sha256 "6e07be0fab5ca1d9c2d9e177718a018fc666141f594a5d6e7025658620cf660a"
+  url "https://www.colm.net/files/ragel/ragel-6.10.tar.gz"
+  sha256 "5f156edb65d20b856d638dd9ee2dfb43285914d9aa2b6ec779dac0270cd56c3f"
 
   bottle do
     cellar :any_skip_relocation
-    rebuild 1
```

Comment:
Leave the `bottle` block untouched, we take care of that when merging the pull request.